features/kalibro_configuration/hotspot_metric_configurations.feature

```diff
@@ -0,0 +1,12 @@
+Feature: HotspotMetricConfiguration retrieval
+  In order to be able to list just with the HotspotMetricConfigurations
+  As a developer
+  I want to get all HotspotMetricConfigurations of a given KalibroConfiguration
+
+  @kalibro_configuration_restart
+  Scenario: get a list of all metric configurations of some kalibro configuration
+    Given I have a kalibro configuration with name "Kalibro for Java"
+    And I have a reading group with name "Group"
```

Comment:
I think hotspot metric configurations do not require reading group associations.